thrift/jaeger.thrift

```diff
@@ -77,6 +77,14 @@ struct Batch {
   2: required list<Span> spans
 }
 
+struct BatchSubmitResponse {
+    1: required bool ok
```

Comment:
Clarify the semantics. What does "not ok" mean? What is the agent expected to do in that case?